### PR TITLE
Address insensitive language

### PR DESCRIPTION
**Pull request by carlisia**
Signed-off-by: Carlisia <carlisia@vmware.com>

### CODE REVIEW

**Comment by carlisia** [comment]
I was able to install Velero using the chart and create a backup.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
       # client version: v2.15.2.
       - name: Install Helm
         run: |
-          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get
+          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get
```
Comment (cpanato) [inline .github/workflows/release.yaml:25]: this returns `404: Not Found`  to me

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: Release Charts
 on:
   push:
     branches:
-      - master
+      - main
```
Comment (cpanato) [inline .github/workflows/release.yaml:6]: i did not see this branch in the current repo, am i missing something?

Comment (nrb) [reply .github/workflows/release.yaml:6]: @carlisia We can make the branch in this repo now and edit this PR.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (nrb) [reply .github/workflows/release.yaml:25]: Yeah, this instance will need to stay as Helm still uses the master branch

Comment (carlisia) [reply .github/workflows/release.yaml:25]: 🤦🏻‍♀️

**Pull request merged**